css/sass/_sponsor.scss

```diff
@@ -134,12 +129,11 @@
         .wcorg-sponsor-level-coffee-break {
             display: flex;
             flex-flow: row wrap;
-
             .wcorg-sponsor {
                 width: calc(100%/3);
                 margin-left: 0;
-                height: auto;;
-
+                height: auto;
+                ;
```

Comment:
@tepkenvannkorn this might unintentionally added semi-colon. Can you check?

Reply:
removed now.